sources/engine/Stride.Physics/Elements/CharacterComponent.cs

```diff
@@ -262,6 +279,7 @@ public void SetVelocity(Vector3 velocity)
         [DataMemberIgnore]
         internal BulletSharp.KinematicCharacterController KinematicCharacter;
 
+        //called after collider shapes are attached to character comp?
```

Comment:
This method is called in PhysicsComponent.Attach() which is called whenever an entity with that component is added to the scene https://github.com/stride3d/stride/blob/master/sources/engine/Stride.Physics/Engine/PhysicsProcessor.cs#L136 
You can remove this comment, and add it as a summary to https://github.com/stride3d/stride/blob/master/sources/engine/Stride.Physics/Engine/PhysicsComponent.cs#L703 instead but it's fine if you don't

Reply:
Good idea, this was a leftover comment I had when I was annotating while learning how this area works. Should have been removed and accidently left it in. I can move that to the PhysicsComponent OnAttach 👍 gives decent context.